python_ta/util/autoformat.py

```diff
@@ -0,0 +1,26 @@
+"""
+This file contains a function for running the Black formatting tool.
+"""
+
+import subprocess
+import sys
+from typing import Optional
+
+
+def run_autoformat(file_path: str, autoformat_options: list[str], max_linelen: Optional[int]):
```

Comment:
Include a return type. Also `max_linelen` should always be an `int`, right? Or did you encounter cases where it was `None`?